src/main/java/org/bytedeco/javacpp/annotation/Virtual.java

```diff
@@ -21,4 +21,5 @@
     /** Pure (abstract) or not. */
     boolean value() default false;
     boolean subclasses() default true;
-}
\ No newline at end of file
+    String javaName() default "";
```

Comment:
I wonder if we can have a better name for that field. Maybe "callback"?

Reply:
Why not. Or `methodName` ? Strictly speaking a java method overriding a c++ method is not really a callback (some function reference we pass as an argument). I agree `javaName` is not great.
What about renaming `value` to `pure` BTW ? I doubt it will break anything since no one used the parameter name `value` when it was the single parameter. Does value make some sense ?

Reply:
> Why not. Or `methodName` ? Strictly speaking a java method overriding a c++ method is not really a callback (some function reference we pass as an argument). I agree `javaName` is not great.

Right, what about just "method"?

> What about renaming `value` to `pure` BTW ? I doubt it will break anything since no one used the parameter name `value` when it was the single parameter. Does value make some sense ?

The name "value" has a special meaning for annotations, we can't change that without breaking compatibility, no.

Reply:
Ah, right.
Commit pushed.